webfront/tests/tests_proteome.py

```diff
@@ -793,52 +691,46 @@ def test_can_get_a_list_from_the_proteome_object(self):
             "/api/proteome/uniprot/UP000006701/entry/interpro/IPR003165/smart",
```

Comment:
I had forgotten that we can filter by integrated member databases. Is it documented somewhere, and is it used by the client?

Reply:
I just checked and couldn't find any mention of it in the docs.
And as far as I remember we don't use this in the website